samples/nrf9160/secure_services/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "Secure Service Sample"
+
+config SAMPLE_SERVICE_RNG
+	bool "Enable code that calls the spm_request_random_number()"
+	default y
+
+# NORDIC_SECURITY_BACKEND is not supported on 53 yet
+	depends on ! SOC_NRF5340_CPUAPP
+
```

Comment:
```suggestion
```

Reply:
This is copied from subsys/spm/Kconfig. It's there to make it noticeable, I think, since we want to remove it eventually.